installation/product-opener.md

```diff
@@ -0,0 +1,16 @@
+# Product Opener Installation
+
+Installation Instructions for Product Opener
+
+### Product Opener Database
+Product Opener is a software used to create and maintain the database of food products in open data. 
+It is released under the GNU Affero License, which is a free copyleft license. This allows Product Opener to be reused by other projects.
+
+### Before you Start
+
+Product opener is NOT yet released as open source. The instructions are for reference and for early testers of Product Opener. These instructions are very likely to change as the code will be restructured, made more independent of proprietary code, and specific Open Food Facts uses etc. When it is released, Product Opener will have much easier to use install scripts and instruction.
```

Comment:
I think this paragraph from the wiki might be outdated. It's released under the AGPL and being developed as FOSS here on GitHub.

Reply:
Sorry about that we fixed the file. We'll also update the wiki to reflect these new changes.